etc/docker/.env.dist

```diff
@@ -0,0 +1,12 @@
+# Symfony application's path (absolute or relative)
+SYMFONY_APP_PATH=./../../../kreta
+SYMFONY_ENV=dev
+
+# MySQL
+MYSQL_ROOT_PASSWORD=root
+MYSQL_DATABASE=kreta
```

Comment:
ok, but one thing is a MySQL connection and other thing is the database itself. We need a kreta database too?